wallet-services/src/main/resources/locale/addon/Wallet_en.properties

```
@@ -451,4 +451,14 @@ exoplatform.wallet.button.addNewPool=Add a new pool
 exoplatform.wallet.button.view=View
 
 exoplatform.wallet.warn.networkVersion=Erroneous network, please switch to Polygon
-exoplatform.wallet.warn.selectedAddress=Erroneous account, please select the corresponding one
\ No newline at end of file
+exoplatform.wallet.warn.selectedAddress=Erroneous account, please select the corresponding one
+
+exoplatform.wallet.label.meedsBtnTitle= create Meeds wallet
+exoplatform.wallet.label.internalWallet=Meeds wallet
+exoplatform.wallet.label.createInternalWallet=Ideal for getting started quickly, your Meeds wallet is protected by a password that only you know
+exoplatform.wallet.label.metamask=MetaMask
+exoplatform.wallet.label.createMetamaskWallet=You already have a Metamask account. You can connect it to eXo. Your wallet will then be available and managed directly in Metamask.
+exoplatform.wallet.label.createMetamaskWalletLearnMore=Learn more:
+exoplatform.wallet.metamask.welcomeMessage=Welcome to digital workplace \n\nClick to SignIn and accept digital workplace terms. This actions will not trigger a blockchain transaction and or any Gas fees. \n\nWallet address: \n{0} \n\nNonce: \n{1}
```

We may use the same pattern, WDYT ? 
```suggestion
exoplatform.wallet.label.metamask.welcomeMessage=Welcome to digital workplace \n\nClick to SignIn and accept digital workplace terms. This actions will not trigger a blockchain transaction and or any Gas fees. \n\nWallet address: \n{0} \n\nNonce: \n{1}
```

---

done!